# This is just a wrapper around the move_group functionality

# Request
string robot_name       # "a_bot", "b_bot", "c_bot"
string program_id       # "insertion", "lin_move", "spiral_press"

# Parameters for each of the programs
float64 force_magnitude         # The force that spiral_press applies, in N
string force_direction          # The direction vector of the force (usually X+ or Z-)
float64 max_radius              # in m
geometry_msgs/Vector3 stiffness # For insertion and spiral_press. In N/m. The 
float64 forward_speed           # The speed at which "insertion" moves forward. Slower probably improves precision.
float64 max_insertion_distance  # 
bool peck_mode                  #
---
# Response
bool success
